### ISSUE TITLE
Ignoring some files

### ISSUE THREAD

**Issue by sashahafner** [AU-BCE-EE/ABM#34]
@fdalby the repo presently has some files from RStudio that clutter things up a bit. 
They can be annoying but it is easy to keep a local copy while having Git ignore them, so they don't show up here on the remote repo.
I will add a .gitignore file from here <https://github.com/github/gitignore/blob/main/R.gitignore>.
Then could you do these steps?
1. Delete any files you don't want to save (actually I can delete any .Rhistory files)
2. Move the other files out (probably just .Rproj)
3. Commit
4. Move the file(s) back in
5. Then they should be ignored in any future commits

We can avoid this next time by starting out with a .gitignore file

**Comment by fdalby** [comment]
@sashahafner I tried this. The ABM.RProj uploads again with a new commit though?

**Comment by sashahafner** [comment]
Oops, that was my mistake, sorry.
Now fixed with d16ccbeb01d4b451411e88f8f1203e2a74300a69 
But now I've removed ABM.Rproj, so be sure to make a copy before you pull, and then you can paste it back it without Git noticing.
Or, contents here (it is just a text file):

```
Version: 1.0

RestoreWorkspace: Default
SaveWorkspace: Default
AlwaysSaveHistory: Default

EnableCodeIndexing: Yes
UseSpacesForTab: Yes
NumSpacesForTab: 2
Encoding: UTF-8

RnwWeave: Sweave
LaTeX: pdfLaTeX

BuildType: Package
PackageUseDevtools: Yes
PackageInstallArgs: --no-multiarch --with-keep.source
```

**Comment by fdalby** [comment]
@sashahafner this issue was resolved I think. closing?
